src/sentry/services/hybrid_cloud/user/model.py

```diff
@@ -34,12 +34,13 @@ class RpcAuthenticator(RpcModel):
     config: Any = None
 
 
-class RpcUser(RpcModel):
+class RpcUserProfile(RpcModel):
```

Comment:
I know introducing this new noun ("profile") could be a little confusing. I hope it fits.

I considered naming it something like `RpcMinimalUser`, but that would mess up the semantics of the "is-a" relationship. That is, `RpcUser` extending `RpcMinimalUser` implies "an `RpcUser` is an `RpcMinimalUser`", which is wrong because `RpcUser` is not minimal.

Reply:
Profile seems like a decent name.

Reply:
I think profile is better than minimal. We have also used `Summary` for similar result types like `OrganizationSummary`.